### ISSUE TITLE
Deceptive improvement.

### ISSUE THREAD

**Issue by Fertanima** [libre-tube/LibreTube#1395]
### Describe your suggested feature

To speed up the loading of large videos, for example, download a 1-hour video by dividing it into 3-minute parts.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Fertanima** [comment]
The video is 1 hour long, but it should be uploaded as a 3-minute video.

**Comment by Fertanima** [comment]
Let it be blocked the rest of the time, then after the video is opened, let it be unblocked.

**Comment by Fertanima** [comment]
1, 2, 3 minute videos open quickly.

**Comment by Bnyro** [comment]
Not possible because we use the Android inbuilt download manager
